src/test/java/com/google/api/codegen/testdata/discoveries/go/go_bigquery.v2.json.baseline

```diff
@@ -20,7 +20,7 @@ import (
 
 func main() {
   ctx := context.Background()
-  httpClient, err := google.DefaultClient(ctx, "https://www.googleapis.com/auth/cloud-platform")
+  httpClient, err := google.DefaultClient(ctx, bigquery.BigqueryScope)
```

Comment:
nit: Would golang purists prefer BigQueryScope? I don't know whether there's an easy way to handle this generally, so I'm fine treating a multi-word API name as a single word for Go naming purposes.